cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,81 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('/');
   });
 
-  it('', () => {
+  it('Student Registration Form', () => {
+    cy.debug('[crossorigin="CORS"]');
+    cy.get('form').get('input[placeholder="First Name"]').type('testName');
+    cy.get('form').get('input[placeholder="Last Name"]').type('testLastName');
+    cy.get('form').get('input[id="userEmail"]').type('test@test.pl');
+    cy.get('form').get('input[id=userNumber]').type('500000100');
+    cy.get('.subjects-auto-complete__value-container')
+      .type('English{enter}');
+    cy.get('.subjects-auto-complete__value-container')
+      .type('Arts{enter}');
+
+    // Select Gender
+    cy.get('input[type="radio"]')
+      .get('.custom-control-label').contains('Male').click();
+
+    // Select Hobbies
+    cy.get('input[type="checkbox"]')
+      .get('.custom-control-label').contains('Sports').click();
+    cy.get('input[type="checkbox"]')
+      .get('.custom-control-label').contains('Music').click();
+
+    // Set DateOfBirthday
+    cy.get('input[id="dateOfBirthInput"]').click();
+    cy.get('.react-datepicker__month-select').select('April');
+    cy.get('.react-datepicker__year-select').select('2024');
+    cy.get('.react-datepicker__day--015').click();
+
+    // Current Address
+    cy.get('textarea[placeholder="Current Address"]').type('Testowa 15, Test');
+    cy.get('#state > .css-yk16xz-control > .css-1hwfws3').type('ncr{enter}');
+    cy.get('#city > .css-yk16xz-control > .css-1hwfws3').type('Delhi{enter}');
```

Comment:
Please construct getters according to the checklist provided in the task. Do not use spaces or `>` etc.